static/app/views/insights/sessions/components/tables/releaseTable.tsx

```diff
@@ -0,0 +1,138 @@
+import type {ReactNode} from 'react';
+import {useCallback, useMemo} from 'react';
+import styled from '@emotion/styled';
+import type {Location} from 'history';
+
+import {DateTime} from 'sentry/components/dateTime';
+import type {GridColumnOrder} from 'sentry/components/gridEditable';
+import GridEditable from 'sentry/components/gridEditable';
+import renderSortableHeaderCell from 'sentry/components/replays/renderSortableHeaderCell';
+import useQueryBasedColumnResize from 'sentry/components/replays/useQueryBasedColumnResize';
+import useQueryBasedSorting from 'sentry/components/replays/useQueryBasedSorting';
+import TextOverflow from 'sentry/components/textOverflow';
+import Version from 'sentry/components/version';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+export type SessionHealthItem = {
+  crash_free_sessions: number;
+  date: string;
+  sessions: number;
+  stage: string;
+  version: string;
+};
+
+interface Props {
+  data: SessionHealthItem[];
+  isError: boolean;
+  isLoading: boolean;
+  location: Location<any>;
+  title?: ReactNode;
+}
+
+const BASE_COLUMNS: Array<GridColumnOrder<string>> = [
+  {key: 'version', name: 'version'},
+  {key: 'date', name: 'date created'},
+  {key: 'stage', name: 'stage'},
+  {key: 'crash_free_sessions', name: 'crash free rate'},
+  {key: 'sessions', name: 'total sessions'},
+];
+
+export default function ReleaseTable({data, isError, isLoading, location, title}: Props) {
+  const {currentSort, makeSortLinkGenerator} = useQueryBasedSorting({
+    defaultSort: {field: 'date', kind: 'desc'},
+    location,
+  });
+
+  const {columns, handleResizeColumn} = useQueryBasedColumnResize({
+    columns: BASE_COLUMNS,
+    location,
+  });
+
+  const renderHeadCell = useMemo(
+    () =>
+      renderSortableHeaderCell({
+        currentSort,
+        makeSortLinkGenerator,
+        onClick: () => {},
+        rightAlignedColumns: [],
+        sortableColumns: [],
+      }),
+    [currentSort, makeSortLinkGenerator]
+  );
+
+  const renderBodyCell = useCallback((column: any, dataRow: any) => {
+    const value = dataRow[column.key];
+    switch (column.key) {
+      case 'date':
+        return <DateTime date={value} />;
+      case 'version':
+        return (
+          <VersionWrapper>
+            <StyledVersion version={value} tooltipRawVersion anchor={false} />
+          </VersionWrapper>
+        );
+      case 'crash_free_sessions':
+        return `${value.toFixed(2)}%`;
+      case 'stage':
+      case 'sessions':
+      default:
+        return <TextOverflow>{value}</TextOverflow>;
+    }
+  }, []);
```

Comment:
For body cell rendering you might have an easier time if you delegate to default field renderers. You can see some [examples in Insights](https://github.com/getsentry/sentry/blob/d19ddaa49c51dd15e12bceea99c4ccdb241d1e14/static/app/views/insights/http/components/tables/domainsTable.tsx#L178-L208) where we specify _custom_ renderers for some table cell types, but then call `getFieldRenderer` for the rest

There are default renderers for `date`, `release`, and others that might be useful!